src/storage/object/copy.js

```diff
@@ -24,24 +24,39 @@ function buildInput(org, key) {
   };
 }
 
-const copyFile = async (client, org, sourceKey, details) => {
+export const copyFile = async (client, daCtx, sourceKey, details, isRename) => {
```

Comment:
does `isRename` Make sense? Maybe more of `includeMetadata`? it flips the flag, but it's more contextually accurate.

Reply:
I think its a matter of taste 😉  I actually had it called `copyMetadata` before but thought it was more natural to express what the purpose of it was than how it was implemented.